add-chain/.env.test

```diff
@@ -0,0 +1,10 @@
+# Required for both Standard Chains and Frontier Chains
```

Comment:
Should this file live in the testdata dir?

Reply:
Fixed in https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/cb6864a042dd45cc0615de28402658cecd361bf3